components/ErrorCustom/index.vue

```diff
@@ -35,7 +39,7 @@
               >
                 {{
                   errorMessage?.statusCode === 404
-                    ? "Halaman tidak ditemukan / Anda tidak memiliki akses untuk halaman ini."
+                    ? "Mohon maaf halaman yang ingin Anda tuju tidak dapat kami temukan."
                     : " Sedang terjadi gangguan pada sistem, silahkan mencoba kembali dan tunggu beberapa saat."
                 }}
               </span>
```

Comment:
### Discussion 
Hello @rayfajars , In my opinion it's be better make in function, so you shouldn't create condition. Or do you have other opinion??

Reply:
i fixed [here](https://github.com/jabardigitalservice/super-app-web/pull/71/commits/2c7020be12cedbd861244ae8f73e946a8ec8541f)